### PR TITLE
chore(early-access-management): Early access management draft mode

### PR DESCRIPTION
**Pull request by EDsCODE**
## Problem

- unclear that when you save an early access feature it will immediately be available for beta usage

## Changes
- add a draft stage that the feature will exist in immediately after initial save. Then provide a button for explicit promotion to beta release
- also removes promotion as suggested [here](https://github.com/PostHog/posthog/pull/15786) (will handle merge conflicts if approved)
Screenshot 2023-05-31 at 1 05 58 PM

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/scenes/early-access-features/earlyAccessFeatureLogic.ts [hunk 1]
```diff
@@ -36,6 +36,7 @@ export const earlyAccessFeatureLogic = kea<earlyAccessFeatureLogicType>([
         cancel: true,
         editFeature: (editing: boolean) => ({ editing }),
         promote: true,
```
Comment (neilkakkar) [inline frontend/src/scenes/early-access-features/earlyAccessFeatureLogic.ts:38]: remove the promote action too?

Comment (EDsCODE) [reply frontend/src/scenes/early-access-features/earlyAccessFeatureLogic.ts]: done

**Review by neilkakkar — APPROVED** [review]
didn't try it locally, but makes sense

**Review by neilkakkar — APPROVED** [review]
didn't try it locally, but makes sense

**Review by fuziontech — APPROVED** [review]
Lgtm

File: posthog/migrations/0318_alter_earlyaccessfeature_stage.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0317_batch_export_models"),
+    ]
+
+    operations = [
+        migrations.AlterField(
```
Comment (neilkakkar) [inline posthog/migrations/0318_alter_earlyaccessfeature_stage.py:13]: while you're doing this migration, might want to add the archived option support as well?

**Pull request merged**